src/main/java/Console.java

```diff
@@ -53,6 +64,56 @@ public void multiArgCommand(String[] input) throws IOException {
 			} else {
 				System.out.println(i + " does not exist.");
 			}
+		} else if (input[0].equals("save")){
+			boolean overWrite = false;
+			// build up string again to allow spaces in file save
+			ArrayList<String> easier = new ArrayList<String>();
+			for (String i : input){
+				easier.add(i);
```

Comment:
Using a StringBuilder here allows us to append each item in the list rather than creating a new String object each time - see - https://www.geeksforgeeks.org/string-vs-stringbuilder-vs-stringbuffer-in-java/

Reply:
I see this is located in other parts of the code as well, I won't leave a comment on each one, this can be directed at them all.